templates/docs/examples/patterns/forms/default.html

```diff
@@ -1,9 +1,8 @@
 {% extends "_layouts/examples.html" %}
-{% block title %}Forms / Dark{% endblock %}
+{% block title %}Forms / Default{% endblock %}
```

Comment:
We probably don't need this example now. It was created to cover dark theme of various elements. All of them are likely already covered by other form examples.